src/Microsoft.DotNet.Build.Tasks/PackageFiles/PerfTesting.targets

```diff
@@ -2,8 +2,7 @@
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <RunPerfTestsForProject Condition="'$(Performance)' == 'true' and '$(RunPerfTestsForProject)' != 'false' and '$(OS)' == 'Windows_NT' and Exists('$(MSBuildProgramFiles32)\MSBuild\Microsoft\Portable\v5.0\Microsoft.Portable.CSharp.targets')" >true</RunPerfTestsForProject>
```

Comment:
Okay, I understand now. I'll echo what I said in the corefx PR, if you have to have a special project configuration for the perf tests, they could probably just go into a separate project file, IMO. I'm not really sure we need to be doing that at all, though, can't we just always include all of the source files in the project? Or am I missing some obvious issue with that?